### ISSUE TITLE
"č" character doesn't work in hints

### ISSUE THREAD

**Issue by svenko99** [brookhong/Surfingkeys#2115]
## Error details

Having a character "č"  (`api.Hints.setCharacters('asdfghjklčweruiocm');`) shows in hints to open a link, but when I press the combination where "č" is present, the link doesn't open.

SurfingKeys: 1.15.0

Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/17.4.1 Safari/605.1.15

URL:  Every URL

## Context

Normal usage

**Comment by Fred-Vatin** [comment]
I confirm this.

### setup
- Firefox 131.0.3, SK 1.16.1
- Vivaldi 6.9.3447.54, SK 1.17.3
- OS Win 10 pro 22H2

### issue
I have a US ansi keyboard. But I use a layout that send `é` when pressing `KeyK`
![image](https://github.com/user-attachments/assets/924427be-d690-434e-bf5c-9fd255cc40bf)

Using
```js
Hints.setCharacters("fdsjél");
```
will show `É` in hints but pressing `KeyK` won’t activate it as expected.

I have to use this settings to actually use `KeyK`:
```js
Hints.setCharacters("fdsjkl");
```
But the hints will show `K` instead of `É`.

### Conclusion
In other words, hints must be set using `event.code` instead of `event.key`. Why not if it is documented. It makes the keys consistant throughout different layouts. But there should be a setting to chose between using `event.key` or `event.code` to display characters in hints.

Vimium C has this settings
![image](https://github.com/user-attachments/assets/593a20af-4b2c-4311-a92e-186bc9bb0f55)
